daemon/top_windows.go

```diff
@@ -0,0 +1,11 @@
+package daemon
```

Comment:
I'm pretty sure that I asked before why you writing tags everywhere even if there is file postfix and you said that it's convention. Did you stopped to use it?